### ISSUE TITLE
Jest Unexpected token export issue

### ISSUE THREAD

**Issue by tat757** [michael-brade/LaTeX.js#93]
I'm trying to use jest to run unit test, but I'm keep getting this error.
```
 FAIL  tests/unit/AboutConsole.spec.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    D:\projects\pqcloud-frontend\node_modules\latex.js\dist\latex.esm.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){var commonjsGlobal="undefined"!=typeof globalThis?globalThis:"undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{};function unwrapExports(t){return t&&t.__esModule&&Object.prototype.hasOwnProperty.call(t,"default")?t.default:t}function createCommonjsModule(t,e){return t(e={exports:{}},e.exports),e.exports}var he=createCommonjsModule(function(t,e){!function(r){var n=e,i=t&&t.exports==n&&t,a="object"==typeof commonjsGlobal&&commonjsGlobal;a.global!==a&&a.window!==a||(r=a);var o=/[\uD800-\uDBFF][\uDC00-\uDFFF]/g,s=/[\x01-\x7F]/g,l=/[\x01-\t\x0B\f\x0E-\x1F\x7F\x81\x8D\x8F\x90\x9D\xA0-\uFFFF]/g,u=/<\u20D2|=\u20E5|>\u20D2|\u205F\u200A|\u219D\u0338|\u2202\u0338|\u2220\u20D2|\u2229\uFE00|\u222A\uFE00|\u223C\u20D2|\u223D\u0331|\u223E\u0333|\u2242\u0338|\u224B\u0338|\u224D\u20D2|\u224E\u0338|\u224F\u0338|\u2250\u0

    SyntaxError: Unexpected token export

      59 | </style>
      60 | <script>
    > 61 | import { parse, HtmlGenerator } from 'latex.js'
         | ^
      62 | export default {
      63 |   name: 'Tutorial',
      64 |   data() {

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:403:17)
      at src/views/tutorial/index.vue:61:1
      at Object.<anonymous> (src/views/tutorial/index.vue:213:3)
```

Here is my ```jest.config.js```
```
,
  transform: {
    '^.+\\.vue$': 'vue-jest',
    '.+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$': 'jest-transform-stub',
    '^.+\\.js$': 'babel-jest'
  },
  transformIgnorePatterns: [
    '/node_modules',
    '/!node_modules\\/latex.js/',
    '/node_modules/(?!_latex.js@0.12.1@latex.js)',
    '/node_modules/(?!latex.js)'
  ],
```
in ```.babelrc```
```
"plugins": [
    "transform-es2015-modules-commonjs",
    "transform-vue-jsx",
    "transform-runtime"
  ]
```
I have tried every solution I could find, but it still not work. Do you have any suggestion what should I do?

**Comment by michael-brade** [comment]
Good idea to try Jest! I have not used it yet, so I don't know how it works but the error message seems clear to me: Jest cannot load LiveScript source, so I guess you have to use something like [this](https://www.npmjs.com/package/livescript-jest) in your jest config.

If you get it to work this way, I would be interested in your solution. You could, for instance, submit a pull request :-)

**Comment by michael-brade** [comment]
I have just released v0.12.3. This should fix any of the import errors you had. And I also say that I misunderstood your report - sorry about that.
